components/camel-azure-storage-datalake/src/test/java/org/apache/camel/component/azure/storage/datalake/integration/DataLakeProducerIT.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.azure.storage.datalake.integration;
+
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Locale;
+
+import com.azure.storage.file.datalake.models.FileSystemItem;
+import org.apache.camel.component.azure.storage.datalake.DataLakeConstants;
+import org.apache.camel.component.azure.storage.datalake.DataLakeOperationsDefinition;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.assertj.core.api.Assertions;
```

Comment:
While I was trying the branch, `assertj -core` wasn't added to the `pom.xml`

Reply:
Good catch, thanks for noticing! I did some experiments with the Azure BOM and I apparently reverted too much when cleaning up.